addons/website_event_booth_sale/models/product_template.py

```diff
@@ -1,12 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import api, models
-
-
-class ProductTemplate(models.Model):
-    _inherit = 'product.template'
-
-    @api.model
-    def _get_product_types_allow_zero_price(self):
-        return super()._get_product_types_allow_zero_price() + ["event_booth"]
```

Comment:
This has higher chances of breaking eCom flows
Like before if I add any product with zero price and this method return detailed type then it let me add product now I can't
For Ex Before this I can add Event product from eCom with Zero price but now I can't and it'll raise an error stating you can't add zero rupee product

Reply:
FYI
This method does called now for events so removing this method for event, still this case happen for booth and course I guess
[PR that remove adding zero price product in cart and creating order line](https://github.com/odoo/odoo/pull/158754)

Need to check with SM PO if we can do same for booth and course :thinking: